### PR TITLE
Document batching settings for Dynatrace

### PR DESCRIPTION
**Pull request by dyladan**
Recommended batching settings for the Dynatrace exporter to avoid being throttled by the server.

### CODE REVIEW

File: exporter/dynatraceexporter/README.md [hunk 1]
```diff
@@ -37,6 +41,9 @@ receivers:
 
 processors:
   batch:
+    # Batch size must be less than or equal to 1000
+    send_batch_max_size: 1000
```
Comment (arminru) [inline exporter/dynatraceexporter/README.md:45]:
```suggestion
    send_batch_size: 1000
```

According to this, `send_batch_max_size` is only supported for the trace pipeline:
https://github.com/open-telemetry/opentelemetry-collector/blob/68c5961f7bc2d8d55fd431ab07367fc16f8b26d0/processor/batchprocessor/README.md#L18-L25

Comment (dyladan) [reply exporter/dynatraceexporter/README.md:45]: gah you're right. Looks like it was introduced here https://github.com/open-telemetry/opentelemetry-collector/pull/1310/files and is still enforced that way

**Comment by dyladan** [comment]
This depends on https://github.com/open-telemetry/opentelemetry-collector/pull/2422 which I am still working on

**Comment by dyladan** [comment]
https://github.com/open-telemetry/opentelemetry-collector/pull/2422 is merged. This PR depends on it being released and the contrib updated to the released version.

File: exporter/dynatraceexporter/README.md (hunk 1 above)
Comment (arminru) [reply exporter/dynatraceexporter/README.md:45]: obsolete, metrics support added in https://github.com/open-telemetry/opentelemetry-collector/pull/2422

**Review by arminru — APPROVED** [review]

**Review by bogdandrutu — APPROVED** [review]

**Pull request merged**